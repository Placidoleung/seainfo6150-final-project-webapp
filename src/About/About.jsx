import React, { useState } from "react";
import styles from './About.module.css';
import Form from "../Form/Form";
import NavBar from "../NavBar/NavBar.jsx";

const About = (props) => {
    return (
        <div className={styles.container}>
            <NavBar/>
            <div className={styles.logo}>
                <a>About BlockNews</a>
            </div>
            <div className={styles.text}>
                <a>BlockNews is a blockchain news website designed by Penghao Liang.</a>
            </div>
        </div>
    )
}

export default About