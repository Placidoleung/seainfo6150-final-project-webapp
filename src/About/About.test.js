import React from 'react'
import About from './About'

describe("About tests", () => {

    it('render correctly ', () => {
        const { container } = render(<About />);
        expect(container).toMatchSnapshot();
    });

});