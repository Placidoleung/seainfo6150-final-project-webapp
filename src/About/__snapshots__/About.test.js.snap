// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About tests render correctly  1`] = `
<div>
  <div
    class="container"
  >
    <div
      class="container"
    >
      <div
        class="logo"
      >
        <a
          href="/"
        >
          BlockNews
        </a>
      </div>
      <ul
        class="nav"
      >
        <li
          class="item"
        >
          <a
            href="/"
          >
            Home
          </a>
        </li>
        <li
          class="item"
        >
          <a
            href="/contact"
          >
            Contact
          </a>
        </li>
        <li
          class="item"
        >
          <a
            href="/about"
          >
            About
          </a>
        </li>
      </ul>
    </div>
    <div
      class="logo"
    >
      <a>
        About BlockNews
      </a>
    </div>
    <div
      class="text"
    >
      <a>
        BlockNews is a blockchain news website designed by Penghao Liang.
      </a>
    </div>
  </div>
</div>
`;
